spec/features/log_as_admin_and_create_admin_user.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+feature "Successful login" do
+  before :each do
+    @user = AdminUser.create!(email: "admin@example.com", password: "password", password_confirmation: "password")
+  end
+
+  background do
+    visit "/admin"
+  end
+
+  scenario "Title has login" do
+    expect(page).to have_content("Login")
```

Comment:
This and a few another scenarios are login functionality tests.

Login is another feature that should have individual spec file to be tested